state_container_py/store.py

```diff
@@ -1,20 +1,36 @@
+from copy import deepcopy
+from typing import Type
+
 from state_container_py.action import Action
 from state_container_py.app_state import ApplicationState
+from state_container_py.reducer import Reducer
+from state_container_py.utils import compose_reducers
 
 
 class Store:
 
     def __init__(self, reducers):
-        # TODO: Deal with defaults
-        self._current_state = ApplicationState()
-
         self._inputReducers = reducers
         self._reducer_instances = self._construct_instance(reducers)
+        self._current_state = self._get_initial_state(self._reducer_instances)
+        self._reduce_method = compose_reducers(self._reducer_instances)
+
+    def get_state(self):
+        return self._current_state
 
     def dispatch(self, action: Action):
```

Comment:
We'll probably need to create a queue to ensure we're running dispatch calls one at a time. We want to apply each state change singularly before moving on to the next.